content/ita/careers.md

```diff
@@ -12,17 +12,22 @@ twitterlargeimage: "/images/careers/banner.png"
 
 ![Facile.it - lavora con noi!](/images/careers/banner.png)
 
-**Facile.it** è in costante crescita e siamo sempre alla ricerca di programmatori da aggiungere al nostro team. Questa proposta è rivolta a sviluppatori di grande esperienza, che credono nell’innovazione e sono sempre attenti alle nuove tecnologie e non vedono l’ora di metterle in pratica. Siamo un gruppo di lavoro dinamico, giovane e divertente. Siamo in totale quasi 80 programmatori e gestiamo numerosi progetti con differenti stack tecnologici.
+**Facile.it** is constantly growing and we are always looking for programmers to join our team.
+This proposal is aimed at highly experienced developers who believe in innovation and are always attentive to new technologies and can't wait to put them into practice. We are a dynamic, young and fun team.
```

Comment:
This proposal is aimed at experienced developers who believe in innovation and are always on the lookout for new technologies and can't wait to put them to work.